server/src/main/java/org/opensearch/cluster/decommission/DecommissionService.java

```diff
@@ -481,4 +482,51 @@ public void onFailure(Exception e) {
             }
         };
     }
+
+    public void deleteDecommissionAttribute(final ActionListener<AcknowledgedResponse> listener) {
+        decommissionController.clearVotingConfigExclusion(new ActionListener<Void>() {
```

Comment:
Can we add a comment here why we need exclusion again?


For abandoned requests, we might not really know if it actually restored the exclusion list. And can land up in cases where even after recommission, exclusions are set(which is unexpected). And by definition of OpenSearch - Clusters should have no voting configuration exclusions in normal operation. Once the excluded nodes have stopped, clear the voting configuration exclusions with DELETE /_cluster/voting_config_exclusions. And hence it is safe to remove the exclusion if any. User should make conscious choice before decommissioning awareness attribute.

Reply:
Done.